src/OrleansProviders/Streams/Memory/MemoryStreamQueueGrain.cs

```diff
@@ -1,57 +1,37 @@
-﻿using System;
+﻿
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Orleans.Providers.Streams.Memory
+namespace Orleans.Providers
 {
     /// <summary>
     /// Memory stream queue grain. This grain works as a storage queue of event data. Enqueue and Dequeue operations are supported.
     /// the max event count sets the max storage limit to the queue.
     /// </summary>
     public class MemoryStreamQueueGrain : Grain, IMemoryStreamQueueGrain
     {
-        /// <summary>
-        /// Type name. Default to the grain name.
-        /// </summary>
-        public const String TypeName = "MemoryStreamQueueGrain";
-
-        /// <summary>
-        /// Event queue. 
-        /// </summary>
-        private Queue<MemoryEventData> eventQueue = new Queue<MemoryEventData>();
+        private readonly Queue<MemoryMessageData> eventQueue = new Queue<MemoryMessageData>();
```

Comment:
The field is private, so it shouldn't generate warnings...  I'll double check.  We don't tend to add comments for private fields unless they are unclear.  This field seems clear, but that's just my opinion.  If you disagree, I'll happily add the comment back.